src/main/java/com/exasol/projectkeeper/ExcludedFilesMatcher.java

```diff
@@ -26,7 +26,7 @@ public ExcludedFilesMatcher(final Collection<String> excludedFilePatterns) {
     }
 
     private static String removeTrailingSlash(final String fileName) {
-        if (fileName.startsWith(FILE_SYSTEM.getSeparator())) {
+        if (fileName.startsWith("/")) {
```

Comment:
In that case, it is indeed intended that it is always the `/`.
Reason: The input is a configuration from the pom file. Since the pom file is always the same I decided that we're using forward slashed there.